samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1703,6 +1703,41 @@
         ]
       }
     },
+    "SSMParameterWritePolicy": {
+      "Description": "Gives access to write values to a parameter in this account. If not using default key, KMSEncryptPolicy will also be needed.",
+      "Parameters": {
+        "ParameterName": {
+          "Description":"The name of the secret to be stored in SSM in your account."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:DescribeParameters"
+            ],
+            "Resource": "*"
```

Comment:
Why "*" on Resource? Can we scope this?

Reply:
We cannot; action `ssm:DescribeParameters` supports all resources, i.e., does not support anything other than "*".

See https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-access.html and attached screenshot from the IAM console.
<img width="1190" alt="Screen Shot 2021-04-16 at 14 19 50" src="https://user-images.githubusercontent.com/11539622/115067523-0ed95b80-9ebf-11eb-994a-5dcd8d21fef1.png">
<img width="1190" alt="Screen Shot 2021-04-16 at 14 19 50" src="https://user-images.githubusercontent.com/11539622/115067541-1567d300-9ebf-11eb-978d-d0a9af37da8b.png">

Reply:
Honestly, I didn't check. Just going through my `Resource: "*"` radar.